tests/integration/test_store.py

```diff
@@ -1881,3 +1881,74 @@ def test_span_extraction_with_ddm_missing_values(
     }
 
     spans_consumer.assert_empty()
+
+
+def test_span_reject_invalid_timestamps(
```

Comment:
Do we need an integration test for this? Could it be a unit test instead?

Reply:
A unit test is not as solid. The timestamp range on spans is only enforced in stand-alone spans and not on spans processed as part of transactions, meaning testing a single component is not enough and we should test the span processing pipeline instead.